cli/common.go

```diff
@@ -129,7 +131,32 @@ func createRequest(slices []string) (controller.Request, error) {
 	return req, nil
 }
 
-func printStatus(groupStatus []fleet.UnitStatus) error {
-	fmt.Printf("%#v\n", groupStatus)
+var (
+	statusHeader = "Group | Units | FDState | FCState | SAState | IP | Machine"
```

Comment:
For me it makes no sense what `fleetctl` is using. For this reason I chose more logic names. IMO this is more descriptive and intuitive. 
- `FDState` -> Fleet Desired State
- `FCState` -> Fleet Current State
- `SAState` -> Systemd Active State